src/Arcus.Messaging.ServiceBus.Core/Extensions/MessageExtensions.cs

```diff
@@ -35,11 +35,12 @@ public static TProperty GetUserProperty<TProperty>(this Message message, string
         ///     Gets the correlation information for a given message.
         /// </summary>
         /// <param name="message">The received message.</param>
-        public static MessageCorrelationInfo GetCorrelationInfo(this Message message)
+        /// <param name="transactionIdPropertyName">Name of the property to determine the transaction id.</param>
+        public static MessageCorrelationInfo GetCorrelationInfo(this Message message, string transactionIdPropertyName = PropertyNames.TransactionId)
```

Comment:
We'll have to be careful that this doesn't break dependent librarires. When a library only updates this core package and not the other packages, we may get a `MethodNotFound` exception along the way.

Reply:
I've split them!